build_all.sh

```diff
@@ -8,11 +8,9 @@
 
 set -eux
 
-#compiler=gnu
-
 target=${target:-"NULL"}
```

Comment:
Hold on. You changed the default to use GNU. AND to build using the debug option.

Reply:
Gah, that was never meant to make it in to a commit. My apologies. I'll open a PR to fix that momentarily.

Reply:
> Hold on. You changed the default to use GNU. AND to build using the debug option.

My caps lock got stuck. Did not mean to yell.